test/clojurewerkz/machine_head/client_test.clj

```diff
@@ -13,10 +13,13 @@
 
 (def ci? (System/getenv "CI"))
 
+(defn connect-high-traffic []
+  (mh/connect "tcp://127.0.0.1:1883" {:opts {:max-inflight 1000}}))
```

Comment:
Please rename this to just `connect` and use a value of 50K or so. There is no downside to it (at least for CI) that I can think of.